test/dataframe.jl

```diff
@@ -1105,6 +1105,11 @@ end
     @test last(df, 6) == DataFrame(A=5:10)
     @test last(df, 1) == DataFrame(A=10)
 
+    @test first(df, 6, true) isa SubDataFrame
```

Comment:
als add tests that the returned data frame has correct contents when `view=true`.